### PR TITLE
[netcore] Implement Assembly.GetExecutingAssembly with stack mark parameter

### PR DESCRIPTION
**Pull request by filipnavara**

### CODE REVIEW

File: mono/metadata/icall.c [hunk 1]
```diff
@@ -5518,6 +5518,17 @@ ves_icall_System_Reflection_RuntimeMethodInfo_GetMethodBodyInternal (MonoMethod
 	return mono_method_body_get_object_handle (mono_domain_get (), method, error);
 }
 
+#if ENABLE_NETCORE
+MonoReflectionAssemblyHandle
+ves_icall_System_Reflection_Assembly_GetExecutingAssembly (MonoStackCrawlMark *stack_mark, MonoError *error)
+{
+	error_init (error);
```
Comment (vargaz) [inline mono/metadata/icall.c]: error_init () is not needed.

Comment (filipnavara) [reply mono/metadata/icall.c]: Right, will remove it.

**Comment by filipnavara** [comment]
@monojenkins build Linux AArch64 Coop Suspend

**Comment by filipnavara** [comment]
@monojenkins build Linux AArch64

**Comment by filipnavara** [comment]
@monojenkins build Linux x64 Coop Suspend

**Pull request merged**